app/views/departure/drafts/DashboardView.scala.html

```diff
@@ -0,0 +1,58 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.components.Heading
+@import viewModels.drafts.AllDraftDeparturesViewModel
+@import views.html.components.DraftDeparturesTable
+@import config.FrontendAppConfig
+
+@this(
+        mainTemplate: MainTemplate,
+        draftDeparturesTable: DraftDeparturesTable,
+        heading: Heading,
+        config: FrontendAppConfig
+)
+
+@(viewModel: AllDraftDeparturesViewModel)(implicit request: Request[_], messages: Messages)
+
+@mainTemplate(
+    title = viewModel.title,
+    showBackLink = true,
+    fullWidth = true
+) {
+
+    @heading(viewModel.heading)
+
+    @draftDeparturesTable(
+        dataRows = viewModel.dataRows,
+        visuallyHiddenHeader = viewModel.visuallyHiddenHeader,
+        rowHeadingReferenceNumber = viewModel.referenceNumber,
+        rowHeadingDaysToComplete = viewModel.daysToComplete
+    )
+
+    <article class="ticket-panel" aria-label="Sub navigation 1">
```

Comment:
Can this be pulled out into a separate component with an a11y spec? Or can we have an a11y spec for this view?